pkgs/development/python-modules/querystring_parser/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "querystring_parser";
+  version = "1.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "644fce1cffe0530453b43a83a38094dbe422ccba8c9b2f2a1c00280e14ca8a62";
+  };
+
+  propagatedBuildInputs = [
+    six
+  ];
+
+  checkPhase = "python querystring_parser/tests.py";
+  # one test fails due to https://github.com/bernii/querystring-parser/issues/35
```

Comment:
you just want to run unit tests, you shoudl be able to exclude integration tests

Reply:
There are no integration tests, only unit tests:  https://github.com/bernii/querystring-parser/blob/master/querystring_parser/tests.py